neuromation/cli/job.py

```diff
@@ -243,6 +250,19 @@ def job() -> None:
     help="File with environment variables to pass",
     secure=True,
 )
+@option(
+    "--life-span",
+    type=str,
+    metavar="TIMEDELTA",
+    help=(
+        "Optional job run-time limit in the format '1d 2h 3m 4s' "
+        "with or without spaces, some parts may be missing. "
+        "Set '0' to disable. "
+        "(default value can be changed in the user config)"
+    ),
+    default=DEFAULT_JOB_LIFE_SPAN,
```

Comment:
Drop `default`. Use `None` as a marker for a case when life-span is not explicitly provided by a user.
Use either config or application-level default then.

Reply:
ok